### ISSUE TITLE
[Suggestion] make collapse button visiable on reading

### ISSUE THREAD

**Issue by Wsine** [levito/tt-rss-feedly-theme#102]
When reading the long article, we may find it not interesting, thus we want to collapse the current article and read another.

In the current CSS setting, the collapse button locates at the top of the title. we need to scroll a long distance to trigger it.

It would be better if the button is still visible during reading.

**Comment by levito** [comment]
Hey, thanks for the suggestion, this sounds like a great idea! I'll give it a try in the next couple of days.

**Comment by levito** [comment]
Hi @Wsine, I just tried it. Unfortunately the only way I see to make this happen is to have the header stick to the top earlier so the collapse bar keeps being visible.

To keep things consistent between layouts, I'd rather not do this by default. But you can add a custom style via the Customize option: 
```css
.cdm.expandable.active .header { top: 30px !important }
```
Use 40px if you use the _cozy_ variant, 27px for _compact_. Does this work for you?

**Comment by Wsine** [comment]
thanks @levito. I tried customizing the style and it is easy. I did following thing:

```css
#content-wrap#content-wrap .cdm .collapse { position: sticky }
```

However, another problem occurs that the page didn't scroll back to the item when it is being collapsed. I think maybe we need to customize both the style and the js logic, and I haven't figure out that.

**Comment by levito** [comment]
Yeah, this would be necessary, so only possible with an additional plugin. However, this is out of scope for this theme, so I'll close this issue. Sorry that I couldn't help, but again, thanks for the suggestion!

**Comment by levito** [comment]
Hey @Wsine, I did a lot of improvements to the theme in the last couple of days, also just addressed this issue. The collapse icon is now visible in the sticky header. However, just like the default theme, it doesn't restore the scroll position. Please feel free to give it a try and let me know how you like it!
